MapboxCoreNavigation/RouteController.swift

```diff
@@ -758,10 +762,10 @@ extension RouteController: CLLocationManagerDelegate {
         // Bearings need to normalized so when the `finalHeading` is 359 and the user heading is 1,
         // we count this as within the `RouteControllerMaximumAllowedDegreeOffsetForTurnCompletion`
         if let upcomingStep = routeProgress.currentLegProgress.upComingStep, let finalHeading = upcomingStep.finalHeading, let initialHeading = upcomingStep.initialHeading {
-            let initialHeadingNormalized = wrap(initialHeading, min: 0, max: 360)
-            let finalHeadingNormalized = wrap(finalHeading, min: 0, max: 360)
-            let userHeadingNormalized = wrap(location.course, min: 0, max: 360)
-            let expectedTurningAngle = differenceBetweenAngles(initialHeadingNormalized, finalHeadingNormalized)
+            let initialHeadingNormalized = Turf.wrap(initialHeading, min: 0, max: 360)
+            let finalHeadingNormalized = Turf.wrap(finalHeading, min: 0, max: 360)
+            let userHeadingNormalized = Turf.wrap(location.course, min: 0, max: 360)
+            let expectedTurningAngle = Turf.differenceBetweenAngles(initialHeadingNormalized, finalHeadingNormalized)
```

Comment:
Maybe this method could be in an extension on CLLocationDirection or CLHeading. 🤔

Reply:
Or `<T: FloatingPoint>` 🤔 ?

Reply:
Or `<T: FloatingPoint>`? 🤔

Reply:
That sounds reasonable, although extending something standard like `FloatingPoint` for something as commonplace as wrapping could run the risk of conflicts with other libraries or application code.